diff --git a/index.d.ts b/index.d.ts
index 7fdd943f637d826f01576ae877f9aa03784a1990..fadf025d905c1bc77de84a11646b51dfe56bb1c1 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -42,6 +42,13 @@ export declare class CompletionClient extends DifyClient {
     stream?: boolean,
     files?: File[] | null
   ): Promise<any>;
+
+  runWorkflow(
+    inputs: any,
+    user: User,
+    stream?: boolean,
+    files?: File[] | null
+  ): Promise<any>;
 }
 
 export declare class ChatClient extends DifyClient {
@@ -63,7 +70,7 @@ export declare class ChatClient extends DifyClient {
 
   getConversations(user: User, first_id?: string | null, limit?: number | null, pinned?: boolean | null): Promise<any>;
 
-  renameConversation(conversation_id: string, name: string, user: User): Promise<any>;
+  renameConversation(conversation_id: string, name: string, user: User, auto_generate?: boolean): Promise<any>;
 
   deleteConversation(conversation_id: string, user: User): Promise<any>;
 }
